library/src/scala/tasty/reflect/Internal.scala

```diff
@@ -119,7 +119,7 @@ import scala.quoted.QuoteContext
  *
  *  ```
  */
-trait Kernel {
+trait Internal {
```

Comment:
`internal` as package names are common -- they are not informative as trait name. What about `CompilerInterface` ?

Reply:
Sounds good